docs/Upgrade-Guide/upgrade_to_4.1.md

```diff
@@ -42,18 +43,23 @@ This procedure involves upgrading **one server at a time**, while keeping the vo
     #gluster volume status
 ```
 
-7. Self-heal all gluster volumes by running
+7. Start glustereventsd service (if it was enabled before), either using the command below, or through other means,
+```sh
+    #systemctl start glustereventsd
+```
+
+8. Self-heal all gluster volumes by running
```

Comment:
Invoke self-heal on all the gluster volumes by running

Reply:
Done